internal/executor/build/task.go

```diff
@@ -0,0 +1,95 @@
+package build
+
+import (
+	"fmt"
+	"github.com/cirruslabs/cirrus-ci-agent/api"
+	"github.com/cirruslabs/cirrus-cli/internal/executor/instance"
+	"strconv"
+	"time"
+)
+
+type TaskStatus int
+
+const (
+	StatusNew TaskStatus = iota
+	StatusSucceeded
```

Comment:
Let's move it in a separate package since `task.Status = build. StatusSucceeded` looks a bit weird.

Reply:
Indeed! Take a look at https://github.com/cirruslabs/cirrus-cli/pull/6/commits/9536927d0e93c7782b6f885dea6625382831a18f.

The other option is to move the whole `Task` to a separate `task` package, but that would clash a lot more with the variable named `task` across the codebase and would probably require us to rename the import from `task` to `taskpkg` or `btask`.